chapters/it/chapter1/3.mdx

```diff
@@ -0,0 +1,329 @@
+# Transformer, per fare cosa?
+
+<DocNotebookDropdown
+  classNames="absolute z-10 right-0 top-0"
+  options={[
+    {label: "Google Colab", value: "https://colab.research.google.com/github/huggingface/notebooks/blob/master/course/chapter1/section3.ipynb"},
+    {label: "Aws Studio", value: "https://studiolab.sagemaker.aws/import/github/huggingface/notebooks/blob/master/course/chapter1/section3.ipynb"},
+]} />
+
+In this section, we will look at what Transformer models can do and use our first tool from the 🤗 Transformers library: the `pipeline()` function.
```

Comment:
Since this section is still in English, would you like to exclude it from this PR? If yes, you'll also need to remove it from the ToC :)

Reply:
Hi @lewtun, I added chapter1/3 after submitting the pull request to start working on it, I didn't figure it'll be included in the PR.

I'm deleting it straight away, sorry.

Reply:
All should be ok now. Have a good day!